### PR TITLE
feature/ongoing-tournament-playoff-view

### PR DESCRIPTION
**Pull request by pinskii**
Add view for a playoff tournament + stylesheet. Includes TODO comments for when the API is finished.

### CODE REVIEW

File: frontend/src/components/modules/OngoingTournament/PlayoffTournamentView.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        onClick={handleNextRound}
+        disabled={isRoundFinished}
+      >
+        Generate Next Round
+      </Button>
+    </Paper>
+  );
+};
+
+const PlayoffTournamentView: React.FC = () => {
```
Comment (vilbedawg) [inline frontend/src/components/modules/OngoingTournament/PlayoffTournamentView.tsx:114]: I would probably separate the `PlayOffTournament` and `TournamentBracket` components into their own files.

File: frontend/src/components/modules/OngoingTournament/PlayoffTournamentView.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        </div>
+      ))}
+      <Button
+        variant="contained"
+        color="primary"
+        onClick={handleNextRound}
+        disabled={isRoundFinished}
+      >
+        Generate Next Round
+      </Button>
```
Comment (vilbedawg) [inline frontend/src/components/modules/OngoingTournament/PlayoffTournamentView.tsx:109]: Should the be disabled once there is only final match left? Right now you can just generate infinitely :D

**Review by vilbedawg — APPROVED** [review]
Left a few comments but other than that looks good.

File: frontend/src/components/modules/OngoingTournament/PlayoffTournamentView.tsx (hunk 2 above)
Comment (pinskii) [reply frontend/src/components/modules/OngoingTournament/PlayoffTournamentView.tsx:109]: Well yes, but in the final product there shouldn't be a button at all, the data to generate new rounds should be drawn from the API which then also knows to stop in the final match

Comment (vilbedawg) [reply frontend/src/components/modules/OngoingTournament/PlayoffTournamentView.tsx:109]: That's true

**Pull request merged**